### PR TITLE
Fix insufficient account with preloaded db files

### PR DESCRIPTION
**Pull request by shiqizng**
Not all accounts from genesis.json were added to the db file when created the initBlock. This caused pingpong to return the following error,

```
ensure accounts failed no accounts found with sufficient stake (> 100000)
Error preparing accounts for transfers: no accounts found with sufficient stake (> 100000)
```
This PR fixes this issue.

### CODE REVIEW

File: netdeploy/remote/deployedNetwork.go [hunk 1]
```diff
@@ -357,34 +357,33 @@ func (cfg DeployedNetwork) GenerateDatabaseFiles(fileCfgs BootstrappedNetwork, g
...
+		if err != nil {
+			return fmt.Errorf("failed to unmarshal %s address %w", alloc.Comment, err)
+		}
+		if comment == srcWalletName {
+			src = addr
+		} else if comment == "feesink" {
+			poolAddr = addr
+		} else if comment == "rewardspool" {
+			sinkAddr = addr
+		}
```
Comment (tsachiherman) [inline netdeploy/remote/deployedNetwork.go:379]: nit: refactor into switch statement

File: netdeploy/remote/deployedNetwork.go [hunk 2]
```diff
@@ -357,34 +357,33 @@ func (cfg DeployedNetwork) GenerateDatabaseFiles(fileCfgs BootstrappedNetwork, g
...
-	if rewardsPool.Address == "" {
-		return fmt.Errorf("error finding source rewards ppol address")
-	}
+	srcWalletName := strings.ToLower(fileCfgs.SourceWalletName)
+
+	for _, alloc := range genesis.Allocation {
+		comment := strings.ToLower(alloc.Comment)
+		addr, err = basics.UnmarshalChecksumAddress(alloc.Address)
+		if err != nil {
+			return fmt.Errorf("failed to unmarshal %s address %w", alloc.Comment, err)
```
Comment (tsachiherman) [inline netdeploy/remote/deployedNetwork.go:371]: nit: you might want to specify the actual address in the error.
i.e.
fmt.Errorf("failed to unmarshal '%s' address '%v' : %w", alloc.Comment, alloc.Address, err)

**Review by tsachiherman — APPROVED** [review]
looks good. thanks for the changes.

File: netdeploy/remote/deployedNetwork.go [hunk 3]
```diff
@@ -357,34 +357,35 @@ func (cfg DeployedNetwork) GenerateDatabaseFiles(fileCfgs BootstrappedNetwork, g
...
+		if err != nil {
+			return fmt.Errorf("failed to unmarshal '%s' address '%v' %w", alloc.Comment, alloc.Address, err)
+		}
+		switch comment {
+		case srcWalletName:
+			src = addr
+		case "feesink":
+			poolAddr = addr
+		case "rewardspool":
+			sinkAddr = addr
```
Comment (cce) [inline netdeploy/remote/deployedNetwork.go:379]: @gmalouf just noticed this is flipping sinkAddr and poolAddr, though in scenario1/2/3 and bootstrappedScenario they are the same address

**Pull request merged**